### DISCUSSION TITLE
Can you specify download resolution?

### DISCUSSION THREAD

**Discussion by will-simp-for-feet-pics** [Avnsx/fansly#93] [Q&A] [answered]
A lot of times I find myself wanting to download something in 1080p when the highest resolution is 4k and that's what the scraper downloads by default. This is annoying because 4k content takes up a lot of space on my hard drive and it takes a long time to download, and I don't even have a 4k display anyway. Is there a way to get the scraper to download content at a lower resolution?

**Comment by Avnsx** [comment, marked as answer]
No there is currently not, star the repository and I'll add the feature

**Reply by will-simp-for-feet-pics** [reply]
aight, done.

**Reply by Avnsx** [reply]
Nevermind I forgot that the fansly API, is too buggy and never responds with correct values for height and width in first place, which made it very complex to even find the heighest resolution media.

https://github.com/Avnsx/fansly-downloader/blob/046961917bf2e5f2e3e0896176160e54699b9840/fansly_downloader.py#L663-L683

The feature you suggested would in-fact be good to have, but I personally won't add it until fansly fixes their API, else it will be very annoying to add this

But this shouldn't be a big problem with version 0.4, because you will only download media once and then hashing (deduplication), from that point on will know, that you already have previously downloaded the media and you won't need to download it again :)

Regarding file-size stored on disk; how bout you don't archive half of fansly, surely then it won't take that much space. Or just buy more drives cba

**Reply by will-simp-for-feet-pics** [reply]
I saw that in the code lol. Oh well, I'll probably just use ffmpeg to downscale video while (hopefully) retaining decent enough quality.
